test/penalizer/Penalizer.test.ts

```diff
@@ -307,3 +365,44 @@ contract('Penalizer', function ([relayOwner, relayWorker, relayManager, otherAcc
     return [rr, sig]
   }
 })
+
+function getSecondsSinceUnixEpoch (): number {
+  return Math.floor(Date.now() / 1000)
+}
+
+async function assertTransactionFails (rawTx: string, reason: string): Promise<void> {
```

Comment:
thanks for adding this function :+1: do you think we could go ahead and use it for the unsuccessful `should receive claims` tests too?

Reply:
@antomor 

i've created a [PR](https://github.com/antomor/rif-relay/pull/1) with this change; please check it out (and change and merge it as you see fit)

Reply:
> @antomor
> 
> i've created a [PR](https://github.com/antomor/rif-relay/pull/1) with this change; please check it out (and change and merge it as you see fit)

Thanks a lot, that saved me time 🙏🏽